spec/mailers/confirmation_instructions_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 require 'rails_helper'
 
-RSpec.describe 'Confirmation Instructions' do
-  describe :notify do
+RSpec.describe 'Devise::Mailer' do
```

Comment:
Are we testing Devise::Mailer here, doesn't seem appropriate,

Reply:
the underlying class being tested here is Devise::Mailer , there is no confirmation mailer available in the code base. 

```
confirmation_mail = Devise::Mailer.confirmation_instructions(confirmable_user.reload, nil, {})
```